crates/sncast/tests/e2e/show_config.rs

```diff
@@ -128,3 +128,40 @@ async fn test_show_config_no_url() {
         wait_timeout: 500
     "});
 }
+
+#[tokio::test]
+async fn test_show_config_default_block_explorer() {
```

Comment:
This check can be added in the `test_show_config_from_snfoundry_toml`  instead of creating a separate test